cranelift-codegen/src/ir/framelayout.rs

```diff
@@ -0,0 +1,73 @@
+//! Frame layout item changes.
+
+use crate::ir::entities::Inst;
+use crate::isa::RegUnit;
+use std::boxed::Box;
+
+#[cfg(not(feature = "std"))]
+use hashmap_core::HashMap;
```

Comment:
```suggestion
use crate::HashMap;
```

This should work both before and after my PR.